packages/zipkin/src/batch-recorder.js

```diff
@@ -105,8 +105,8 @@ class BatchRecorder {
 
   _writeSpan(id, span, isNew = false) {
     // TODO(adriancole) refactor so this responsibility isn't in writeSpan
-    if (!isNew && this.partialSpans.get(id) === 'undefined') {
-      // Span not found.  Could have been expired.
+    if (!isNew && this.partialSpans.get(id) === undefined) {
```

Comment:
my guess is this meant 'typeof' as undefined checks are a bit dodgy

```suggestion
    if (!isNew && typeof this.partialSpans.get(id) === 'undefined') {
```

Reply:
Done.